### PR TITLE
updated graph.py

### PR DESCRIPTION
**Pull request by Virat-Shrimali**
updated graph.py file with the requested changes in issue #22

### CODE REVIEW

File: package/graph.py [hunk 1]
```diff
@@ -1,33 +1,74 @@
...
+    Raises:
+        ValueError: If the password is incorrect.
+        FileNotFoundError: If the snippet file is not found.
+    """
+    # Retrieve the current time in HHMM format
+    current_time = datetime.now().strftime("%H%M")
+    
+    # Check if the provided password matches the current time
+    if str(password) != current_time:
+        raise ValueError("Incorrect password.")
```
Comment (this-is-yaash) [inline package/graph.py:55]: Don't mention anything that's related to password in the print statement.
Use placeholder words instead, like `syntax error` or `logical error`.
or anything that's related to python's default error cases.

**Comment by this-is-yaash** [comment]
@Virat-Shrimali please make the changes based on the review comments I provided.

**Review by this-is-yaash — COMMENTED** [review]
I have noted four review changes; please make the necessary updates.

File: package/graph.py (hunk 1 above)
Comment (Virat-Shrimali) [reply package/graph.py]: Thankyou for this detailed reviews, I have made the said changes. If anything else is need to be changed, please inform me.

**Comment by Wahid7852** [comment]
Can you do the changes after syncing with the source?
do a git pull and then make a PR please
or do a fresh clone, do your changes and PR please!

**Pull request closed without merging**